.gitignore

```diff
@@ -15,5 +15,6 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.idea
```

Comment:
no, it's relative and will help with your next contributors. I'm using WebStorm and it's automatically creating some .idea files which become a part of any project.